openedx/adg/lms/applications/serializers.py

```diff
@@ -0,0 +1,147 @@
+"""
+Serializers for applications app
+"""
+
+from django.utils.translation import ugettext_lazy as _
+from rest_framework import serializers
+
+from .models import Education, WorkExperience
+
+# pylint: disable=translation-of-non-string
+
+COMPLETED_DATE_YEAR = 'date_completed_year'
+COMPLETED_DATE_MONTH = 'date_completed_month'
+
+
+class EducationSerializer(serializers.ModelSerializer):
+    """
+    Education serializer
+    """
+
+    def validate(self, attrs):
```

Comment:
How about renaming `attrs` to `data` as it is more relevant and DRF also use data for this.

Reply:
The overridden name is attrs, we get pylint warning for different overridden name, so instead to suppress this warning. I changed variable name.